src/baseWrapper.ts

```diff
@@ -15,6 +18,16 @@ export default class BaseWrapper<ElementType extends Element> {
     this.wrapperElement = element
   }
 
+  abstract find(selector: string): DOMWrapper<Element>
```

Comment:
I really like how we set our expectations on both kind of wrappers (DOMWrapper & VueWrapper) by leveraging abstract methods